### PR TITLE
Fellowship certificate changes

### PR DESCRIPTION
**Pull request by binson-b**

### CODE REVIEW

File: certificate/urls.py [hunk 1]
```diff
@@ -38,8 +38,8 @@
     url(r'^scipy_download_2016/$', 'scipy_download_2016', name='scipy_download_2016'),
     url(r'^openfoam_symposium_download_2016/$', 'openfoam_symposium_download_2016', name='openfoam_symposium_download_2016'),
     url(r'^openfoam_symposium_feedback_2016/$', 'openfoam_symposium_feedback_2016', name='openfoam_symposium_feedback_2016'),
-    url(r'^fossee_internship_cerificate_download/$', 'fossee_internship_cerificate_download', name='fossee_internship_cerificate_download'),
-    url(r'^fossee_internship16_cerificate_download/$', 'fossee_internship16_cerificate_download', name='fossee_internship16_cerificate_download'),
+    url(r'^fossee_internship15_cerificate_download/$', 'fossee_internship_cerificate_download', name='fossee_internship15_cerificate_download'),
+    url(r'^fossee_internship_cerificate_download/$', 'fossee_internship16_cerificate_download', name='fossee_internship_cerificate_download'),
```
Comment (prathamesh920) [inline certificate/urls.py]: I see inconsistency in the names, which slightly confuses, also same views for two URLs but different name!

Comment (binson-b) [reply certificate/urls.py]: I will comment the 41st line, since we are not using it..is that ok?

Comment (prathamesh920) [reply certificate/urls.py]: Simply remove it.

Comment (binson-b) [reply certificate/urls.py]: okay

File: certificate/templates/fossee_internship_cerificate_download.html [hunk 2]
```diff
@@ -16,7 +16,7 @@ <h4 class="modal-title" id="invalidModalLabel">Invalid Category</h4>
                 </div>
               </div>
           </div>
-          <form class="col-lg-12" action="{% url 'certificate:fossee_internship_cerificate_download' %}" method="post">
+          <form class="col-lg-12" action="{% url 'certificate:fossee_internship15_cerificate_download' %}" method="post">
```
Comment (prathamesh920) [inline certificate/templates/fossee_internship_cerificate_download.html]: Now, this URL reverse match will fail!
As fossee_internship**15**_cerificate_download does not exist.

File: certificate/views.py [hunk 3]
```diff
@@ -2437,26 +2438,28 @@ def fossee_internship_cerificate_download(request):
     return render_to_response('fossee_internship_cerificate_download.html', context, ci)
 
 
-def create_fossee_internship_cerificate(certificate_path, name, qrcode, type, paper, internship_project_duration, student_edu_detail, student_institute_detail, superviser_name_detail, workshop, file_name):
+def create_fossee_internship_cerificate(certificate_path, name, qrcode, wtype, paper, internship_project_duration, student_edu_detail, student_institute_detail, superviser_name_detail, workshop, file_name):
```
Comment (prathamesh920) [inline certificate/views.py]: Since you are changing, can this be as per PEP8 standards!

**Review by prathamesh920 — CHANGES_REQUESTED** [review]
Kindly resolve comments.
After resolving re-check every component related.

File: certificate/templates/fossee_internship_cerificate_download.html (hunk 2 above)
Comment (binson-b) [reply certificate/templates/fossee_internship_cerificate_download.html]: this file will never be called as I have removed the source from index.html and urls.py

File: certificate/views.py (hunk 3 above)
Comment (binson-b) [reply certificate/views.py]: okay, i will do that

File: certificate/views.py [hunk 4]
```diff
@@ -2437,26 +2438,34 @@ def fossee_internship_cerificate_download(request):
...
+      certificate_path, name, qrcode,
+      wtype, paper, internship_project_duration, student_edu_detail,
+      student_institute_detail, superviser_name_detail,
+      workshop, file_name):
     error = False
     try:
         download_file_name = None
-        year = internship_project_duration[internship_project_duration.find('to')-5:internship_project_duration.find('to')].strip()
-        if type == 'P':
+        year = internship_project_duration[
```
Comment (prathamesh920) [inline certificate/views.py:2449]: I hope this is fine and will work in all the cases!
For now I am ignoring this but see to it that it's generic and implemented in a more better way.
If that is the case(if change needed) then only create an issue for the same.

**Review by prathamesh920 — APPROVED** [review]

File: certificate/views.py (hunk 4 above)
Comment (binson-b) [reply certificate/views.py:2449]: ok will create a issue

**Pull request merged**